src/firebase-db.ts

```diff
@@ -61,6 +62,19 @@ const configurations: IConfigs = {
   }
 };
 
+const MAX_FIRESTORE_SAVE_TIME = 1000;
```

Comment:
One second seems short to me, but I don't have much confidence in my intuition here. I guess we can see how it behaves in the field. We don't want to spam users with flaky networks with repeated alerts.

Reply:
Right, it was there just to make some manual tests easier. :)